scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -11,18 +11,17 @@
             [plumbing.core :as pc])
   (:import (com.google.auth.oauth2 GoogleCredentials)
            (io.kubernetes.client ApiClient)
-           (io.kubernetes.client.apis CoreV1Api)
            (io.kubernetes.client.util Config)
            (java.io FileInputStream File)
            (java.util UUID)
            (java.util.concurrent Executors ScheduledExecutorService TimeUnit)))
 
 (defn generate-offers
   "Given a compute cluster and maps with node capacity and existing pods, return a list of offers."
-  [node-name->node pod-name->pod compute-cluster]
+  [node-name->node pod-name->pod starting-pod-name->pod compute-cluster]
```

Comment:
Do we want to reorder this to put compute-cluster first? (that seems more 'normal'.

Reply:
Done